src/components/composer/composer/composer.ts

```diff
@@ -107,8 +108,9 @@ interface ComposerState {
 
 interface AutoCompleteState {
   showProvider: boolean;
-  selectedIndex: number;
+  selectedIndex: number | undefined;
   values: AutocompleteValue[];
+  type: "function" | "dataValidation";
```

Comment:
should be formula, not function

Reply:
we can argue it's auto-completing a function (within a formula)